osh/hub/scripts/osh-retention

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: Copyright contributors to the OpenScanHub project.
+
+"""
+Script for cron that performs the retention policy
+"""
+import datetime
+import logging
+import os
+import shutil
+from datetime import date
+
+os.environ['DJANGO_SETTINGS_MODULE'] = 'osh.hub.settings'
+
+
+def deleteScansResults(scansToDelete):
+    from kobo.hub import models
+    for scan in scansToDelete:
+        try:
+            pathToDelete = models.Task.get_task_dir(scan.id)
+            if os.path.isdir(pathToDelete):
+                shutil.rmtree(pathToDelete)
+        except OSError as e:
+            logging.error(f"Error deleting scans: {e}")
+
+
+# In order to identify personal scans, we are using the owner field. If the owner doesn't contain any / or @, we assume
+# that is a personal scan
+def identifyPersonalScan(owner):
+    # We could use regexps here if we want to make this more complex but for this specific case
+    # built-in string methods are enough and we are more perfomance-oriented.
+    if "/" in owner or "@" in owner:
+        return False
+    else:
+        return True
```

Comment:
This whole expression can be simplified as
```python
return "/" not in owner and "@" not in owner
```
which is rather short for a separate function.  Thus, it can be easily inlined to the calling function.

Reply:
In the new version, I used your return statement but I would like to keep the function as it helps to identify which rules are we checking. In the future when we have more rules, It would be good to be able to identify them quickly and add more cases